L1Trigger/DTTriggerPhase2/interface/MPThetaMatching.h

```diff
@@ -0,0 +1,71 @@
+#ifndef Phase2L1Trigger_DTTrigger_MPThetaMatching_h
+#define Phase2L1Trigger_DTTrigger_MPThetaMatching_h
+
+#include "L1Trigger/DTTriggerPhase2/interface/MPFilter.h"
+
+#include <iostream>
+#include <fstream>
+
+// ===============================================================================
+// Previous definitions and declarations
+// ===============================================================================
+
+// ===============================================================================
+// Class declarations
+// ===============================================================================
+
+class MPThetaMatching : public MPFilter {
+public:
+  // Constructors and destructor
+  MPThetaMatching(const edm::ParameterSet &pset);
+  ~MPThetaMatching() override;  // = default;
+
+  // Main methods
+  void initialise(const edm::EventSetup &iEventSetup) override;
+  void run(edm::Event &iEvent,
+           const edm::EventSetup &iEventSetup,
+           std::vector<cmsdt::metaPrimitive> &inMPaths,
+           std::vector<cmsdt::metaPrimitive> &outMPaths) override;
+  void run(edm::Event &iEvent,
+           const edm::EventSetup &iEventSetup,
+           std::vector<cmsdt::metaPrimitive> &allMPaths,
+           std::vector<cmsdt::metaPrimitive> &inMPaths,
+           std::vector<cmsdt::metaPrimitive> &outMPaths) override {};
+  void run(edm::Event &iEvent,
+           const edm::EventSetup &iEventSetup,
+           MuonPathPtrs &inMPath,
+           MuonPathPtrs &outMPath) override {};
+
+  void finish() override;
+
+  // Public attributes
+
+  float vwire = 24.4;                                    // cm/ns
+  float zFE[5] = {-658.9, -393.3, 126.4, 393.3, 658.9};  //cm
+  float xFE[3] = {218 / 2., 266.8 / 2., 315 / 2.};       //cm
+  float ZRES_CONV = 65536. / 1500;
```

Comment:
From @quinnanm: is it possible to add some details in the comment as to what these numbers are please?

Reply:
Correct. I added a description and moved them to L1Trigger/DTTriggerPhase2/interface/constants.h where they fit better since they are constexpr.
Thank you @quinnanm !